### PR TITLE
Setting a chirp-mass limit on the mchirp reweighting

### PR DESCRIPTION
**Pull request by spxiwh**
So I basically want to be able to choose a limit on the mchirp when doing mchirp reweighting in the new OGC statistic class. Although I want this to be optional, and I want the variable to be the user's choice.

But I wanted to discuss first how best to implement this, and how to provide it on the command line. I would think to provide this through `**kwargs` to the stat function. But should we add a `--stat-kwargs max_mchirp:40 some_other_value:20` command line option to `coinc_findtrigs`? ... That seems to be the best way to do this IMO.

### CODE REVIEW

**Comment by spxiwh** [comment]
This might also start to simplify the stat function, as we *could* use one statistic class, but (for example) choose the new_snr ranking as a kwarg option to the stat. `--stat-kwargs max_mchirp:40 sngl_stat:newsnr_sgveto_with_denty_bins`. (Not something I'll include here, but just a thought)

**Comment by tdent** [comment]
should I look at this?

**Comment by ahnitz** [comment]
@spxiwh Sorry, I saw [WIP] and didn't get passed that to read your question. In any case, yes, I think your proposal for providing keyword arguments seems reasonable. I'm not sure of any better way to do that.

**Comment by spxiwh** [comment]
@ahnitz WIP tag now removed. This has been implemented as suggested. I think this can be used to greatly reduce the number of unique stat classes in play, but I'll leave that for a later PR.

**Comment by spxiwh** [comment]
The codeclimate issues are not new, but coming from similar code still in place (I hope that would be resolved when cleaning this up more in the future). I think that unused `**kwargs` is not a problem to ensure the `__init__` functions all take the same arguments.

File: bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs [hunk 1]
```diff
@@ -29,6 +29,12 @@ parser.add_argument("--statistic-files", nargs='*', action='append', default=[],
 parser.add_argument("--ranking-statistic", choices=stat.statistic_dict.keys(),
                     default='newsnr',
                     help="The coinc ranking statistic to calculate")
+parser.add_argument("--statistic-key-word-arguments", nargs='*',
```
Comment (ahnitz) [inline bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs]: seems longer than necessary name. How about just '--statistic-keywords'

Comment (spxiwh) [reply bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs]: Changed as requested,

**Pull request merged**